PyInstaller/hooks/hook-gi.repository.Gio.py

```diff
@@ -0,0 +1,37 @@
+import glob
+import os
+import sys
+
+import PyInstaller.log as logging
+from PyInstaller.compat import is_darwin, is_win
+from PyInstaller.utils.hooks import get_typelibs, exec_statement
+
+logger = logging.getLogger(__name__)
+
+
+hiddenimports = ['gi.overrides.Gio']
+
+
+datas = get_typelibs('Gio', '2.0')
+
+
+binaries = []
+
+statement = """
+from gi.repository import Gio
+print(Gio.__path__)
+"""
+
+path = exec_statement(statement)
+pattern = None
+
+if is_darwin:
+    pattern = os.path.join(os.path.commonprefix([sys.prefix, path]), 'lib', 'gio', 'modules', '*.so')
```

Comment:
Thanks for the explanation. Can you please put this into a comment, so someone else will get it in 5 years. Thanks.